src/javascript.js

```diff
@@ -33,7 +33,8 @@ export default class JavaScriptScanner {
           id: message.ruleId.toUpperCase(),
           message: messages[message.ruleId.toUpperCase()].message,
           description: messages[message.ruleId.toUpperCase()].description,
-          code: message.source,
+          code: message.ruleId.toUpperCase(),
+          eslintCode: message.source,
```

Comment:
This is the source code that caused the error, so a property name like `sourceCode` or `sourceText` would be better.